lib/jellyfish/room.ex

```diff
@@ -92,7 +94,19 @@ defmodule Jellyfish.Room do
   @spec add_component(id(), Component.component(), map()) ::
           {:ok, Component.t()}
           | :error
-          | {:error, :incompatible_codec | :reached_components_limit_hls}
+          | {:error,
+             {:component_disabled_globally, String.t()}
```

Comment:
You are using `globally` naming, does it mean something can be enabled locally only?